tasks/overcloud-setup-load-inventory.yml

```diff
@@ -8,7 +8,8 @@
 - name: import inventory
   shell: |
       source ~/stackrc
-      openstack baremetal import nodes.json
+      openstack overcloud node import --introspect --provide nodes.json
```

Comment:
We're already doing the introspection separately, if we add --introspec should we remove the next task?